src/sinks/humio/logs.rs

```diff
@@ -21,19 +21,28 @@ use crate::{
     tls::TlsConfig,
 };
 
-const HOST: &str = "https://cloud.humio.com";
+pub(super) const HOST: &str = "https://cloud.humio.com";
 
 /// Configuration for the `humio_logs` sink.
 #[configurable_component(sink("humio_logs"))]
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct HumioLogsConfig {
     /// The Humio ingestion token.
+    #[configurable(metadata(
+        docs::examples = "${HUMIO_TOKEN}",
+        docs::examples = "A94A8FE5CCB19BA61C4C08"
+    ))]
     pub(super) token: SensitiveString,
 
-    /// The base URL of the Humio instance.
+    /// The base URL of the Humio instance. The scheme (`http` or `https`) must be specified.
```

Comment:
👍  I do think we have enough "endpoints" and "addresses" throughout the sources and sinks that it would be well worth zeroing in on consistency. I've tried to make the ones I've touched consistent, but would probably be good to get consensus.